Doc/library/pdb.rst

```diff
@@ -639,6 +639,54 @@ can be overridden by the local file.
 
    Print the return value for the last return of the current function.
 
+.. pdbcommand:: exceptions [excnumber]
```

Comment:
We need to add an entry in Doc/whatsnew/3.13.rst  since this is a new feature.

Reply:
Isn't that what `Misc/NEWS.d/next/Library/2023-07-12-10-59-08.gh-issue-106670.goQ2Sy.rst` is for ?

Reply:
Not really. News is for the release manager. What'snew is for users.

Reply:
Ok, I though this was auto-merged into the whatsnew. Added.